src/utils/show-help-message.ts

```diff
@@ -0,0 +1,14 @@
+import chalk from "chalk";
+
+export const showHelpMessage = () => {
+  console.log(` ${chalk.bold.blue("Usage:")}
+    ${chalk.bold.green("npx create-eth<@version>")} ${chalk.gray("[-i | --install | -s | --skip | --skip-install] [-e <extension> | --extension <extension>] [--dev] [-h | --help]")}
+`);
```

Comment:
Ohh also not sure if we should expose `--dev`  flag to end user normal, Since its solely used in local development / creating extension.

Reply:
Thanks! Removed it [250c99d](https://github.com/scaffold-eth/create-eth/pull/63/commits/250c99df4e5236ad230a6a4f9edb7d8b3d9157c4)